files/common/usr/bin/get-hotfix-version

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# Copyright 2020 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -o errexit
+set -o pipefail
+
+function usage() {
+	echo "Usage: $(basename "$0")"
+	echo
+	echo "Display the hotfix version information for the appliance."
+	exit 2
+}
+
+function hotfix_version() {
+	zfs get -Hpo value "com.delphix:hotfix-version" "$(dirname "$(zfs list -Hpo name /)")" | tr -d '\n'
+}
+
+options=$(getopt -l "" -o "" -- "$@")
+eval set -- "$options"
```

Comment:
What benefit do we gain by using `getopt` here? Since we don't support options, would it be more simple to remove this part?